build.sh

```diff
@@ -13,7 +13,7 @@ if [ -d ".git" ]; then
 
 SHA=`git rev-parse --short --verify HEAD`
 DATE=`git show -s --format="%cd" --date=short HEAD`
-REV="$SHA - $DATE"
+REV="$SHA -- $DATE"
```

Comment:
did you plan to add another dash?

Reply:
Yes, so that the commit hash and the date in the version are separated by dash, not hyphen in the PDF. Just a small cosmetic thing.